### DISCUSSION TITLE
Where to find documentation regarding JDK versions supported for each Quarkus release?

### DISCUSSION THREAD

**Discussion by dhoffer** [quarkusio/quarkus#43164] [Community]
We are planning a Quarkus and JDK version upgrade.  Where to find documentation regarding JDK versions supported for each Quarkus release version?

**Comment by cescoffier** [comment]
That's a good question. One solution is to go to https://quarkus.io/guides/getting-started and look at the JDK prerequisites. The Java requirement differs depending on the Quarkus version you select (in the dropdown at the top).

This would be the minimal version. We do not indicate the maximum version, but we do tests with all the non-EA JDK (for example, test with 17, 21, and 22). We even have a few tests with 23-EA (but there are a few limitations).


For an upgrade, I would select the JDK LTS (21 as _we speak_).
